pytradfri/kelvin.py

```diff
@@ -0,0 +1,28 @@
+from .const import (ATTR_LIGHT_COLOR_X as X, ATTR_LIGHT_COLOR_Y as Y)
+
+
+KNOWN_XY = {
+  2200: {X: 33135, Y: 27211},
+  2700: {X: 30140, Y: 26909},
+  4000: {X: 24930, Y: 24694},
+}
+
+
+def can_dekelvinize(kelvins):
+    return min(KNOWN_XY.keys()) <= kelvins <= max(KNOWN_XY.keys())
+
+
+def dekelvinize(kelvins):
```

Comment:
I totally agree.

By the way: I'm still not sure whether we'll need reverse `xy_to_kelvin` function for integration with `home-assistant`.